test-files/test_issue_611_globalthis.ts

```
@@ -17,3 +17,38 @@ const store = (globalThis as any)[storeId] as Map<string, string>;
+  if (loopScriptVar === 0) continue outer;
+  seen.push(loopScriptVar);
+}
+console.log(
+  "labeled loop:",
+  (globalThis as any).loopScriptVar,
+  seen.join(","),
+);
+
+switch (1) {
+  case 1:
+    var switchScriptVar = "case";
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)